src/lib/vina.cpp

```diff
@@ -330,32 +326,62 @@ void Vina::compute_vina_maps(double center_x, double center_y, double center_z,
 		// m_model
 		std::cerr << "ERROR: Cannot compute Vina or Vinardo affinity maps. The (rigid) receptor was not initialized.\n";
 		exit(EXIT_FAILURE);
+	} else if (size_x <= 0 || size_y <= 0 || size_z <= 0) {
+		std::cerr << "ERROR: Grid box dimensions must be superior than 0 Angstrom.\n";
```

Comment:
"superior" -> "greater"

Reply:
Oh god, I am starting to revert to French. Thanks!